### DISCUSSION TITLE
u/jamesL813's logo touchups

### DISCUSSION THREAD

**Discussion by PickleJesus123** [godotengine/godot-proposals#8297] [General Discussions]
Hello everyone,

I noticed a [reddit thread](https://www.reddit.com/r/godot/comments/10wdz4n/redesigned_my_attempt_at_fitting_godots_logo_into/) from 9 months ago where u/jamesL813 made some very minor touchups to the Godot application icon. The touchups were showered in praise by the Godot community. I worry that the reddit thread may have gone unnoticed by the project maintainers, since they aren't super active on the subreddit, so I decided to post it here on the more "official" Github discussions. I think the touchups bring some extra professionalism to the branding, while staying true to Godot's roots. Anyways, here were his minor touchups:
![edits](https://github.com/godotengine/godot-proposals/assets/12555805/6060e5cc-610e-4081-b507-2b6ca5a87a6e)

**Comment by AThousandShips** [comment]
As pointed out elsewhere this kind of minor touchups are unlikely to be accepted because of the amount of work needed to change the logo everywhere, even just official places

Further while there's differing opinions on whether the current logo is good (though the dislike for the current logo is IMO exaggerated) it is going to be essentially impossible to pick *one* out of the many proposals for changing it

The people disliking the current design aren't guaranteed to like one specific new one, as the one thing uniting them is just dislike for the current one

Minor changes are, after all, just personal preference, and therefore opinions will divide

**Reply by AThousandShips** [reply]
Further any discussion is moot unless the original creator has consented to the possibility of them being official (and consent to the logo being licenced in the necessary way, i.e. an CC license)

**Reply by PickleJesus123** [reply]
Hello,
I just want to clear up any confusion. This thread only pertains to u/jamesL813's revisions and is not intended to be an open-ended discussion about the dozens of proposed revisions

**Reply by AThousandShips** [reply]
How does that change the point? The point is: Why should we choose *this* particular one, it is essentially guaranteed to not be popular among a majority of people, please read what I wrote more clearly :)

**Reply by PickleJesus123** [reply]
In my original message up at the top, I mentioned that u/jamesL813's proposed revisions appear to be unanimously showered in praise by the Godot community

**Reply by AThousandShips** [reply]
Except you can't gauge that from just a reddit post and some up/down votes, it has some 440 score, unsure how many up/down votes there are, but that isn't remotely unanimous

**Reply by PickleJesus123** [reply]
Of all the comments made 9 months ago, there isn't a single negative one

**Reply by AThousandShips** [reply]
That doesn't say anything, it just says no one spoke up, there's just 23 comments at all, if there were hundreds or thousands that still wouldn't say anything, but with that few chances are the post simply didn't pull the attention of anyone who felt like commenting

Lack of criticism doesn't mean approval, that's completely backwards

That's the issue I already expressed, the popularity of individual suggestions occur in echo chambers or aren't something that people spend energy on criticising

**Reply by PickleJesus123** [reply]
That's why I'm posting it here - to see what the Github community thinks.

**Reply by AThousandShips** [reply]
Fair enough, but the end result is likely the same, a lack of consensus, even if these kinds of minor tweaks were seriously considered which they generally aren't regardless of popularity, as it's generally not worth the effort

But point remains: it doesn't matter unless the artist actually approves and approves of licensing that's compatible (just copying them to this place doesn't necessarily have their consent, just because it's put on the internet doesn't mean you can just copy and share it)

**Reply by PickleJesus123** [reply]
If it got to that stage, I would happily liaise with the creator and try to get a license

**Reply by AThousandShips** [reply]
No, that's backwards 🙃 

You can't even offer this as an idea for an official one without consulting them first, and copying the logo is breeching their copyright in the first place

You can't just offer *someone else's* art as a suggestion without consulting with them first

**Reply by PickleJesus123** [reply]
I'm not offering anything... I simply linked a reddit post to what everyone thinks. If Github likes it, I'll correspond with the creator and see if they'll allow it to be used in the distribution

**Reply by AThousandShips** [reply]
But you are suggesting and sharing them, without even checking if the original artist wants that, or is okay with that, you don't see how that's an issue?

This is a proposal discussion, this is suggesting this be added, there's no other way to see that

**Reply by PickleJesus123** [reply]
Linking a reddit post does not violate copyright law. The screenshot was taken directly from Google Images.

**Reply by AThousandShips** [reply]
My suggestion:
Get in touch with the artist and ask about this, and then continue, otherwise this should be closed

And consider this in the future

**Reply by AThousandShips** [reply]
> The screenshot was taken directly from Google Images.

That doesn't matter...

You don't seem to consider the aspects of copyright and consent here, which is an issue

**Reply by Zireael07** [reply]
Both of you are kinda right. OP can link a post or a picture if he wants, but the discussion is moot without author's consent and clear copyright status since it can't be added otherwise

**Reply by PickleJesus123** [reply]
I have given up on this discussion, and it will remain closed indefinitely. 

I had heard that Juan wanted Godot to appeal more to studios, judging by his posts since the Unity debacle. I thought that, as head of a small studio whose apps have 100,000,000+ installs, I would weigh in with my initial impressions of Godot. I just want Godot to succeed. I didn't realize that there was such intense (borderline irrational) emotional attachment to the current iteration of the logo. My bad. I give up

**Reply by AThousandShips** [reply]
You're completely ignoring everything I've said I'd you assume that I'm "irrationally attached" to the current icon

**Reply by PickleJesus123** [reply]
The synopsis of your replies has been "1) A logo change would be a lot of work and 2) How could we even know if anyone likes this revision". 

1) I agree with you - a logo change would take work. But it would be worthwhile. You clearly don't agree

2) We don't even know if anyone likes this revision - which is why I posted it here. Has anyone had a chance to say "I like this revision" or "I don't like this revision"? The post received a few likes, but you have been bombarding this discussion since its inception. Nobody had a chance to discuss this particular revision set. A neutral and unbiased person would say "let's see what everyone thinks about this revision". That's why I claim you are being irrational

**Reply by AThousandShips** [reply]
I have not bombarded anything with "I don't like this", I've pointed out practical and theoretical issues, but given your response to the previous discussion you're unable to see constructive feedback as anything but whining or complaining 

Well you assume what ever you want, I'm out, have a good day

**Reply by PickleJesus123** [reply]
Setting our differences aside, I really do appreciate your contributions to Godot. Thank you

**Reply by AThousandShips** [reply]
Thank you, much appreciated

**Comment by jmarceno** [comment]
I just don´t understand why open a new discussion, when you already had #8278 for this exact purpose. 
The suggestions are different, but the subject is exactly the same.
Is this to get more visibility because the new ones are at the top?

**Reply by PickleJesus123** [reply]
My other discussion thread was very vague and open ended. This thread strictly pertain's to u/jamesL813's revisions and nothing else

**Reply by AThousandShips** [reply]
But opening this hides the various points made there, so it's not really logical to close and open a new one instead of changing the focus

**Reply by PickleJesus123** [reply]
Many of the replies in the previous thread were made in response to my extremely vague questions/statements, and the discussion derailed. This thread is about one specific revision

**Reply by AThousandShips** [reply]
The discussions going on there weren't unhelpful at all IMO, they were directly on topic and discussed the issues with the process itself
